packages/content/src/Application/ContentDataMapper/DataMapper/ExcerptDataMapper.php

```diff
@@ -49,31 +50,42 @@ public function map(
     }
 
     /**
-     * @param mixed[] $data
+     * @param array<string, mixed> $data
      */
     private function setExcerptData(ExcerptInterface $dimensionContent, array $data): void
     {
         if (\array_key_exists('excerptTitle', $data)) {
+            Assert::nullOrString($data['excerptTitle']);
```

Comment:
not sure about this "runtime" checks maybe we should go with `assert` instead which is compiled away on `prod` for better performance in cases used for imports.